pycomm3/packets/util.py

```diff
@@ -75,14 +75,29 @@ def wrap_unconnected_send(message: bytes, route_path: bytes) -> bytes:
 
 
 def request_path(
-    class_code: Union[int, bytes],
-    instance: Union[int, bytes],
+    class_code: Union[int, bytes, list],
+    instance: Union[int, bytes, list],
     attribute: Union[int, bytes] = b"",
 ) -> bytes:
-    segments = [
-        LogicalSegment(class_code, "class_id"),
-        LogicalSegment(instance, "instance_id"),
-    ]
+    segments = []
+    if isinstance(class_code, list):
```

Comment:
I think we should change this to check for any sequence types and not limit it just to `list`, like a `tuple` should work as well: `class_code=(0x01, 0x02)`.  

what about using `isinstance(..., Sequence)` from `collections.abc` instead?

Reply:
Sure! That is definitely better.